src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((pair) => pair.split(':').map((str) => str.trim()))
+    .filter(([property, value]) => property && value)
+    .reduce((styles, [property, value]) => {
+      styles[property] = value;
```

Comment:
When setting properties on the `styles` object, consider checking for and removing any potential CSS comments within the `property` or `value` strings, as they could cause the resulting object to have incorrect keys or values.